src/io/flutter/run/daemon/FlutterAppManager.java

```diff
@@ -41,7 +41,7 @@
 
   private static final Gson GSON = new Gson();
 
-  private final Logger myLogger = Logger.getInstance("#io.flutter.run.daemon.FlutterAppManager");
+  private static final Logger LOG = Logger.getInstance(FlutterAppManager.class.getName());
```

Comment:
I'm just changing the logging to match the pattern used in other files. If we do want to keep the logger as an instance logger (and not static), I'm happy to do that...